### PR TITLE
REDIS-36 Fixed HTTP responses and import issues

### PR DESCRIPTION
**Pull request by EngineersBox**
## Changes

* Fixed import issues
* Fixed HTTP response returning from decorator
* Added implicit `flask.request.remote_addr` capture with decorator usage
* Fixed module structure

### CODE REVIEW

File: modules/RateLimiter/src/request_handler.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import functools
 from redis import Redis, ConnectionPool
-from config import CONNECTION, LIMITING
+from .config import LIMITING, CONNECTION
```
Comment (ariadev79) [inline modules/RateLimiter/src/request_handler.py:3]: difference between this and the previous one?

**Review by ariadev79 — COMMENTED** [review]
Just clarrifying

File: modules/RateLimiter/src/request_handler.py (hunk 1 above)
Comment (EngineersBox) [reply modules/RateLimiter/src/request_handler.py:3]: Python requires dot syntax for relative pathing (E.g. `.<filename>`), this just adds that

**Review by ariadev79 — APPROVED** [review]
Done. Small change

**Pull request merged**